packages/clay-core/src/vertical-bar/Panel.tsx

```diff
@@ -60,7 +46,7 @@ export function Panel({children, keyValue, tabIndex}: Props) {
 		<CSSTransition
 			aria-labelledby={`${id}-tab-${keyValue}`}
 			className={classNames('sidebar', {
-				'c-slideout-show': activePanel === keyValue && isFirst,
```

Comment:
@pat270 we have to ignore this in the first rendering, for example when opening the panel it is not doing animation anymore.

Reply:
@matuzalemsteles Yes sort of, React ends up re-rendering which removes `c-slideout-show`.